signing_service/signing_service/tests/test_handle_connection.py

```diff
@@ -195,7 +196,26 @@ def test_that__handle_connection_should_send_error_frame_if_payload_golem_messag
         assertpy.assert_that(deserialized_message.payload[0]).is_equal_to(ErrorCode.UnexpectedMessage)
         assertpy.assert_that(deserialized_message.request_id).is_equal_to(REQUEST_ID_FOR_RESPONSE_FOR_INVALID_FRAME)
 
-    def _prepare_and_execute_handle_connection(self, raw_message, handle_connection_wrapper=None):
+    def test_that__handle_connection_should_continue_loop_when_heartbeat_frame_is_received(self):
+        heartbeat_frame = HeartbeatFrame(
+            payload=None,
+            request_id=777,
+        )
+        raw_message = heartbeat_frame.serialize(private_key=CONCENT_PRIVATE_KEY)
+        with mock.patch('signing_service.signing_service.send_over_stream') as send_mock:
+            with mock.patch('signing_service.signing_service.logger') as logger_mock:
+                response = self._prepare_and_execute_handle_connection(raw_message, expect_response_from_scoket=False)
+
+                assertpy.assert_that(response).is_equal_to(mock.sentinel.no_response)
+                send_mock.assert_not_called()
+                logger_mock.info.assert_not_called()
+
+    def _prepare_and_execute_handle_connection(
+        self,
+        raw_message,
+        handle_connection_wrapper=None,
+        expect_response_from_scoket=True
```

Comment:
please add type declarations

Reply:
we agreed not to include type declarations in tests (they are optional)